src/bower_components/emby-webcomponents/homesections/homesections.js

```diff
@@ -273,35 +273,12 @@
     }
 
     function loadlibraryButtons(elem, apiClient, user, userSettings, userViews) {
+        elem.classList.remove('verticalSection');
+        var html = getLibraryButtonsHtml(userViews);
 
-        return Promise.all([getAppInfo(apiClient), getDownloadsSectionHtml(apiClient, user, userSettings)]).then(function (responses) {
-
-            var infoHtml = responses[0];
-            var downloadsHtml = responses[1];
-
-            elem.classList.remove('verticalSection');
-
-            var html = getLibraryButtonsHtml(userViews);
-
-            elem.innerHTML = html + downloadsHtml + infoHtml;
-
-            bindHomeScreenSettingsIcon(elem, apiClient, user.Id, userSettings);
-
-            if (infoHtml) {
-                bindAppInfoEvents(elem);
-            }
-            imageLoader.lazyChildren(elem);
-        });
-    }
-
-    function bindAppInfoEvents(elem) {
-
-        elem.querySelector('.appInfoSection').addEventListener('click', function (e) {
-
-            if (dom.parentWithClass(e.target, 'card')) {
-                registrationServices.showPremiereInfo();
-            }
-        });
+        elem.innerHTML = html;
+        bindHomeScreenSettingsIcon(elem, apiClient, user.Id, userSettings);
+        imageLoader.lazyChildren(elem);
```

Comment:
I think this needs to return a promise? Quickly tracing the calls it looks like the result of this function is pushed to an array of promises.

Reply:
done, thanks for catching this